Main/CommandLineInterface.cpp

```diff
@@ -216,5 +231,8 @@ int runFromCommandLine(const StringList& arguments, ArmipsArguments settings)
 		return 1;
 	}
 
+	if (settings.showStats)
```

Comment:
Can you move this to ```runArmips```? That way it's available for tests and whatever else links against it too.

Reply:
Sure.  Was thinking it was cli only, but that makes sense.

-[Unknown]